concepts/05 Widgets/Form/20 Change Options at Runtime/15 Editor Options.md

```diff
@@ -58,11 +62,126 @@ With Angular, bind the option to change in the [editorOptions](/api-reference/10
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div>
+            <DxForm :form-data="employee">
+                <DxSimpleItem data-field="firstName" :editor-options="editorOptions" />
+                <DxSimpleItem data-field="lastName" />
+                <DxSimpleItem data-field="phone" />
+                <DxSimpleItem data-field="email" />
+            </DxForm>
+            <DxCheckBox
+                text="Disable the First Name Editor"
+                :value.sync="isFirstNameDisabled"
+                @value-changed="checkBoxValueChanged" />
+        </div>
+    </template>
+    <script>
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import { DxForm, DxCheckBox } from 'devextreme-vue';
```

Comment:
There should be no imports directly from 'devextreme-vue' or 'devextreme-react'. Please check that across all these documents.

Reply:
Fixed